utils/env_setup.sh

```diff
@@ -37,6 +37,15 @@ if [[ $PEANO_INSTALL_DIR == "" ]]; then
   export PEANO_INSTALL_DIR="$(pip show llvm-aie | grep ^Location: | awk '{print $2}')/llvm-aie"
 fi
 
+NPU=`/opt/xilinx/xrt/bin/xrt-smi examine | grep -E "NPU Phoenix|NPU Strix|NPU Strix Halo|NPU Kracken"`
```

Comment:
Should we be sourcing the setup script here instead?  To put xrt-smi in PATH...

Reply:
Suggestion below